src/components/PageWizard/_page-wizard.scss

```diff
@@ -6,7 +6,7 @@
   display: flex;
 
   &__sticky {
-    padding-bottom: 5rem;
+    padding-bottom: $layout-06;
```

Comment:
this token does not map to 5rem. Have we checked the changes that this would make in the UI?

Reply:
There's no value that directly maps to 5rem so I may leave this one as an exception

Reply:
This will not break the layout, so it can be standardized 👍